dart/lib/src/protocol/sentry_user.dart

```diff
@@ -64,33 +74,62 @@ class SentryUser {
   ///
   /// These keys are stored as extra information but not specifically processed
   /// by Sentry.
+  final Map<String, dynamic>? data;
+
+  @Deprecated('Will be removed in v7. Use [data] instead')
   final Map<String, dynamic>? extras;
 
+  /// Approximate geographical location of the end user or device.
```

Comment:
Add a comment that this is automatically inferred by the server if `ipAddress` is set.
Also that we don't collect that automatically since it could be PII sensitive.

Reply:
Added in 3afd2c569777935658e795d683eb1962914f5220